core/src/apps/webauthn/fido_seed.py

```
@@ -0,0 +1,18 @@
+def ensure_fido_seed(func):
+    def wrapper(*args, **kwargs):
+        from trezor.crypto import se_thd89
+        from utime import sleep_ms
+
+        while True:
+            try:
+                ret = se_thd89.fido_seed()
+                if ret:
+                    break
+                else:
```

Ruff (0.8.2)

Unnecessary `else` after `break` statement

Remove unnecessary `else`

(RET508)